pkg/providers/cloudstack/config/template-md.yaml

```diff
@@ -115,6 +115,11 @@ metadata:
     cluster.x-k8s.io/cluster-name: {{.clusterName}}
   name: {{.workerNodeGroupName}}
   namespace: {{.eksaSystemNamespace}}
+  {{- if .autoscalingConfig }}
+  annotations:
+    cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: "{{ .autoscalingConfig.MinCount }}"
+    cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: "{{ .autoscalingConfig.MaxCount }}"
```

Comment:
Has this been tested end to end with an actual CloudStack cluster? Do we need an e2e test added to verify this functionality?